draft-ietf-quic-recovery.md

```diff
@@ -615,6 +615,8 @@ are described in this section.
 bytes_in_flight:
 : The sum of the size in bytes of all sent packets that contain at least
   one retransmittable frame, and have not been acked or declared lost.
+  The size does not include IP or UDP overhead.  Ack only frames do not
+  count towards byte_in_flight.
```

Comment:
> The reason acks shouldn't count towards bytes in flight is one never wants to delay an ack due to lack of congestion window.

You can count acknowledgment-only packets, and it will only cause bytes_in_flight to hit the critical threshold faster.  If you want to send an ACK without paying attention to the congestion window, then you could move the special logic to the sending of a packet:

```
if (!congested) {
  packet.send();
} else if (packet.contains(ACK)) {
  packet.strip_everything_except(ACK);
  packet.send();
}
```

I think that we should probably discuss at some point.  I have no problem with landing something that is more concrete than what we currently have in the meantime.